src/components/Messages/Message/index.js

```diff
@@ -43,6 +44,11 @@ const onClickTranscript = async () => {
 	await handleTranscript();
 };
 
+const canPrintTranscript = () => {
+	const { transcript, enableTranscriptMobile } = store.state.config.settings;
+	return transcript && isMobile() ? enableTranscriptMobile : true;
+};
+
```

Comment:
@AlexanderKanakis When the transcript functionality is disabled (Not mobile transcript) It returns true. So it shows the `Save Transcript` button.

You can try disabling both transcript settings and it still shows the `Save Transcript` button